scripts/terraform/azure/postgresql/init-database.sql

```diff
@@ -0,0 +1,31 @@
+CREATE ROLE ctb_user WITH
+	LOGIN
+	NOSUPERUSER
+	NOCREATEDB
+	NOCREATEROLE
+	INHERIT
+	NOREPLICATION
+	CONNECTION LIMIT -1
+	PASSWORD '--> ENTER PASSWORD HERE <--';
```

Comment:
This is for the user (ie, me or whoever is on the command line) to fill in once the server has been created, right?

Reply:
Yes, this is to actually create a database user so it can connect to the database.

That could have been done by Terraform as well. However, I didn't see any value of doing it because PostgreSQL for PostgreSQL, better stay with the SQL language than switching between Azure resource template and AWS RDS equivalent. I focused on creating the resources from Terraform, and the post-creation is done through SQL scripts.